spec/integration/post_show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show', type: :feature do
+  describe 'Post' do
+    before(:each) do
+      @first_user = User.create(name: 'Tom', photo: 'margaret.jpeg', bio: 'Teacher from Mexico.', posts_counter: 0)
+      @first_user.save!
```

Comment:
- [ ] Please make sure that the tests run successfully and pass. Currently, there're tests that fail, check the attached below.
![image](https://user-images.githubusercontent.com/62290545/181051796-6e42400b-d1fd-4cc7-935a-a371100cc1dc.png)